recipes/rosco/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+mkdir build
+cd build 
+
+cmake \
+    -DCMAKE_Fortran_FLAGS="-ffree-line-length-0" \
```

Comment:
Maybe add `-DCMAKE_BUILD_TYPE=Release` for Unix as well.

Reply:
added